<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemView</class>
 <widget class="QWidget" name="ItemView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SizeConstraint::SetMaximumSize</enum>
   </property>
   <item>
    <layout class="QVBoxLayout" name="_2">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="itemIcon">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>cache/item/Aghanim's Blessing.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="itemTitle">
       <property name="minimumSize">
        <size>
         <width>459</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Title</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="itemDescription">
       <property name="minimumSize">
        <size>
         <width>459</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Description</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
